### ISSUE TITLE
opening the "points of interest" menu from homescreen gives massive red error

### ISSUE THREAD

**Issue by kwietelmann** [calvin-cs262-fall2021-teamH/Client#65]
```js
Warning: Each child in a list should have a unique "key" prop.

Check the render method of `SignedOutLocationList`. See https://fb.me/react-warning-keys for more information.
    in View (at signedOutLocations.js:112)
    in SignedOutLocationList (at SceneView.tsx:126)
    in StaticContainer
    in StaticContainer (at SceneView.tsx:119)
    in EnsureSingleNavigator (at SceneView.tsx:118)
    in SceneView (at useDescriptors.tsx:209)
    in RCTView (at View.js:34)
    in View (at DebugContainer.native.tsx:27)
    in DebugContainer (at NativeStackView.native.tsx:71)
    in MaybeNestedStack (at NativeStackView.native.tsx:229)
    in RNSScreen (at createAnimatedComponent.js:165)
    in AnimatedComponent (at createAnimatedComponent.js:215)
    in ForwardRef(AnimatedComponentWrapper) (at src/index.native.tsx:147)
    in Screen (at NativeStackView.native.tsx:175)
    in SceneView (at NativeStackView.native.tsx:277)
    in RNSScreenStack (at NativeStackView.native.tsx:268)
    in NativeStackViewInner (at NativeStackView.native.tsx:322)
    in RNCSafeAreaProvider (at SafeAreaContext.tsx:76)
    in SafeAreaProvider (at SafeAreaProviderCompat.tsx:46)
    in SafeAreaProviderCompat (at NativeStackView.native.tsx:321)
    in NativeStackView (at createNativeStackNavigator.tsx:67)
    in NativeStackNavigator (at App.js:61)
    in EnsureSingleNavigator (at BaseNavigationContainer.tsx:430)
    in ForwardRef(BaseNavigationContainer) (at NavigationContainer.tsx:132)
    in ThemeProvider (at NavigationContainer.tsx:131)
    in ForwardRef(NavigationContainerInner) (at App.js:60)
    in App (created by ExpoRoot)
    in ExpoRoot (at renderApplication.js:45)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:106)
    in RCTView (at View.js:34)
    in View (at AppContainer.js:132)
    in AppContainer (at renderApplication.js:39)
at node_modules/react/cjs/react.development.js:315:4 in printWarning
at node_modules/react/cjs/react.development.js:287:16 in error
at node_modules/react/cjs/react.development.js:1630:9 in validateExplicitKey
at node_modules/react/cjs/react.development.js:1656:27 in validateChildKeys
at node_modules/react/cjs/react.development.js:1806:23 in createElementWithValidation
at screens/signedOutLocations.js:106:8 in SignedOutLocationList
at screens/signedOutLocations.js:29:23 in fetch.then.then$argument_0

```

**Comment by TheBguy87** [comment]
Fixed in PR #68. Some children in the list still needed keys or unique keys.

**Comment by TheBguy87** [comment]
Resolved
